src/Blueprint/Blueprint/help/Set-AzBlueprintArtifact.md

```diff
@@ -0,0 +1,373 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Blueprint.dll-Help.xml
+Module Name: Az.Blueprint
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blueprint/set-azblueprintartifact
+schema: 2.0.0
+---
+
+# Set-AzBlueprintArtifact
+
+## SYNOPSIS
+Update an artifact in a blueprint.
+
+## SYNTAX
+
+
+### UpdateTemplateArtifact (Default)
+```
+Set-AzBlueprintArtifact -Name <String> -Type <PSArtifactKind> -Blueprint <PSBlueprintBase>
+ [-Description <String>] [-DependsOn <System.Collections.Generic.List`1[System.String]>]
+ -TemplateParameterFile <String> -TemplateFile <String> [-ResourceGroupName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### UpdateArtifactByInputFile
+```
+Set-AzBlueprintArtifact -Name <String> -Blueprint <PSBlueprintBase> -ArtifactFile <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### UpdateRoleAssignmentArtifact
+```
+Set-AzBlueprintArtifact -Name <String> -Type <PSArtifactKind> -Blueprint <PSBlueprintBase>
+ [-Description <String>] [-DependsOn <System.Collections.Generic.List`1[System.String]>]
+ -RoleDefinitionId <String> -RoleDefinitionPrincipalId <String[]> [-ResourceGroupName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### UpdatePolicyAssignmentArtifact
+```
+Set-AzBlueprintArtifact -Name <String> -Type <PSArtifactKind> -Blueprint <PSBlueprintBase>
+ [-Description <String>] [-DependsOn <System.Collections.Generic.List`1[System.String]>]
+ -PolicyDefinitionId <String> -PolicyDefinitionParameter <Hashtable> [-ResourceGroupName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Update an artifact. There are two ways to update an artifact: either through an artifact JSON as an input file or through providing inline parameters for the artifact. 
+while former doesn't require type of the artifact to be provided latter method requires user to provide the type of the artifact through -Type parameter.
```

Comment:
While should be capitalized.  This sentence should be fixed like the other one as former/latter are confusing in context.